staking/params.go

```diff
@@ -0,0 +1,18 @@
+package staking
+
+import (
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+const (
+	isValidatorKeyStr = "Harmony/IsValidator/v0"
+	isValidatorStr    = "Harmony/IsAValidator/v0"
+	isNotValidatorStr = "Harmony/IsNotAValidator/v0"
```

Comment:
Why this is needed? isNotValidatorStr.  If needed, then "Harmony/IsNotAValidator/v0" => "Harmony/IsNotValidator/v0". Otherwise, remove.

Reply:
It's not needed, I will remove it

Reply:
ops, Edgar merged it so fast....